src/single-spa-css.test.ts

```diff
@@ -1,4 +1,5 @@
 import * as singleSpa from "single-spa";
+import { createEmitAndSemanticDiagnosticsBuilderProgram } from "typescript";
```

Comment:
This import doesn't seem to be used. Does it perform some side effect just by being imported?

Reply:
👍  yeah good catch, removed.